src/services/invoice.services.ts

```
@@ -246,6 +261,179 @@ export const voidInvoice = async (merchantId: string, id: string) => {
+      status: InvoiceStatus.PENDING,
+      expiresAt: onChain?.expiresAt ?? null,
+      // Backdated to the ledger close time so a historical replay does not
+      // report every on-chain invoice as created on the day of the replay.
+      createdAt: occurredAt,
+    });
+
+    return {
+      invoice: created,
+      outcome: (candidates.length > 1 ? 'created-ambiguous' : 'created') as
+        | 'created'
+        | 'created-ambiguous',
```

ESLint

[error] Replace `|·'created'⏎·······` with `'created'`

(prettier/prettier)